source/standards/monitoring.html.md.erb

```diff
@@ -1,44 +1,26 @@
 ---
-title: Monitoring
+title: Monitor your service
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Availability monitoring
+At GDS, we follow the principles set out in the Service Manual for [monitoring the status of services](https://www.gov.uk/service-manual/technology/monitoring-the-status-of-your-service) and for [setting performance metrics](https://www.gov.uk/service-manual/measuring-success/how-to-set-performance-metrics-for-your-service).
 
-[Pingdom](https://www.pingdom.com/) is the tool we've used most for availability
-monitoring. If you need to monitor a service from outside the US or EU,
-GOV.UK has used [CA App Synthetic Monitor](https://asm.ca.com/en/).
+Reliability Engineering is currently running an Alpha on the use of [Prometheus](https://prometheus.io/) as our opertional metrics service to GDS, please contact us <a href="mailto:the-gds-way@digital.cabinet-office.gov.uk?subject=feedback">by email</a> if you want to know more.
```

Comment:
`Reliability Engineering is currently running an Alpha on the use of [Prometheus](https://prometheus.io/) as our operational metrics service to GDS. This will be available to all teams using the recommended hosting options. Please contact us <a href="mailto:the-gds-way@digital.cabinet-office.gov.uk?subject=feedback">by email</a> if you want to know more at this stage.`